src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] report;
+        try {
+            report = createReport(fromFileName);
+            writeToFile(toFileName, report[0], report[1], report[2]);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read/write file.", e);
+        }
```

Comment:
```suggestion
       String textFromFile = readFromFile(fromFileName);
        String report = createReport(textFromFile);
        writeToFile(toFileName, report);
```
Задачу треба розбити на підзадачі так, щоб це були окремі незалежні дії із зрозумілими проміжними результатами. Код методу тоді буде наче алгоритм написаний англійською.